Makefile

```
@@ -1,13 +1,13 @@
 NAME          := hdf5
 SRC_EXT       := bz2
-TEST_PACKAGES := $(NAME) java-$(NAME) $(NAME)-devel $(NAME)-static $(NAME)-mpich-tests $(NAME)-openmpi-tests
+TEST_PACKAGES := $(NAME) java-$(NAME) $(NAME)-devel $(NAME)-static $(NAME)-mpich-tests $(NAME)-openmpi3-tests
 
 include packaging/Makefile_packaging.mk
 
 PR_REPOS := 
 
-hdf5_1.10.4+repack-1.debian.tar.xz:
-	curl -f -L -O http://ftp.us.debian.org/debian/pool/main/h/hdf5/hdf5_1.10.4+repack-1.debian.tar.xz
+hdf5_1.12.0+repack-1~exp2.debian.tar.xz:
```

I think you should be able to use:
```suggestion
hdf5_$(VERSION)+repack-1~exp2.debian.tar.xz:
```
here, to help reduce future maintenance burden.